### ISSUE TITLE
Looking for new maintainer(s)

### ISSUE THREAD

**Issue by raniejade** [JetBrains/spek#959]
It was around five years ago when @hhariri reached out to me, to discuss the merge of Spek and [KSpec](https://github.com/raniejade/kspec). Both projects have similar goals (KSpec was born out of my frustration of Spek at that time) so the merge was very beneficial for both projects. Fast forward to now, it has been quite a journey and the experience in maintaining an open source project (Spek was my first big one btw!) helped me grow quite a lot as a software engineer. Sadly, all journeys must come to an end - I'll be stepping down as maintainer. My interests have shifted and there is this burning drive within me to work on something different.

I'm looking for new maintainer(s) that will bring Spek to the next level, if you are interested or know someone that might be - please reach out to me in [Kotlin Slack](kotlinlang.slack.com/) (either in the #spek channel or dm) or twitter: [@raniejade](https://twitter.com/raniejade)

I'll be sticking for a while to provide bugfixes until a new maintainer(s) can be found.

**Comment by sh-soltanpour** [comment]
Hi, I tried to reach out to you using Twitter but it seems that your dm is restricted on Twitter. Please let me message you there (@_shahryar_s).
Thank you.

**Comment by raniejade** [comment]
Hey @sh-soltanpour - just followed you back. That should allow you to DM me :)

**Comment by snijsure** [comment]
@raniejade @sh-soltanpour having a **documented** process of generating new plugin for Android studio would be super useful. If there is such a document I would be happy to test that process on various 4.1.3 , 4.2 (beta) versions of Android studio.

Currently I am finding upgrading to latest version of Android studio is almost impossible because , you can't run Spek tests successfully when using Spek plugin version - **2.0.15-IIJ2020.1**

**Comment by raniejade** [comment]
Yeah, that's a good idea. If you are still interested in building your own version of the plugin you can take a look at https://github.com/spekframework/spek/blob/2.x/spek-ide-plugin-android-studio/build.gradle.kts#L8

**Comment by snijsure** [comment]
@raniejade I will absolutely try this.

However can you help decoding these magic numbers and how you arrive at those numbers?

```
    "AS41" to ij.BuildConfig(
        "**201.7223.91**",
        "Studio4.1",
        "AS34",
        ij.VersionRange("**201.8743.12**", "**201.8743.12.***"),
        arrayOf("android", "java", "**org.jetbrains.kotlin:1.3.72-release-Studio4.1-5"**)
    )
```

**Comment by raniejade** [comment]
@snijsure sorry for the delay - busy with life admin.

The first one `201.7223.91` is the SDK used to build the plugin and second one `ij.VersionRange("**201.8743.12**", "**201.8743.12.***")` is the version the plugin will support. Normally both versions should be the same, however, if we supported the AS version at a non-GA staged (alpha/beta/rc) the first version would be reflecting the IJ version the non-GA AS is based off. To get the number of AS specifically, you need to download AS and look at the about page - the version should be there.

**Comment by raniejade** [comment]
Just an update: Still looking for maintainers to take over Spek - I barely have time to work on personal project nowadays.

**Comment by MetroSteffenPaul** [comment]
I would be interested in maintaining spek. we currently use it for unit testing production code in our company and I am annoyed that it does not work with the latest idea version.
so please count me in :)

**Comment by raniejade** [comment]
@MetroSteffenPaul Thank you! (and apologies for not fixing it - barely have time nowadays) As for your specific problem can you take a look at #988 and backport it to the `2.0.x` branch?

**Comment by MetroSteffenPaul** [comment]
@raniejade no worries, taking a look.

**Comment by ennioVisco** [comment]
Hello, it's been a year and a half since the last message, any update about the project?

**Comment by jaredsburrows** [comment]
@raniejade How can I help? Would be nice to migrate this to kmp.
